tests/roles/mariadb_copy/tasks/env_vars_src.yaml

```diff
@@ -8,8 +8,7 @@
       SOURCE_MARIADB_IP={{ source_mariadb_ip|default(external_mariadb_ip) }}
       declare -A SOURCE_GALERA_MEMBERS
       SOURCE_GALERA_MEMBERS=(
-      ["{{ edpm_node_hostname }}"]={{ source_mariadb_ip|default(external_mariadb_ip) }}
-      # ...
+        {{  source_galera_members }}
       )
```

Comment:
this makes the code and docs looking different...

Reply:
I think we require the bash snippets to be the same, but for setting the variables we can't really make it the same as the docs. We can't hardcode values here like docs have hardcoded examples, we need to feed the values from the environment (ansible vars). E.g. look below at how SOURCE_DB_ROOT_PASSWORD is set.

Or maybe i don't follow what you mean?